cmd/kubeadm/app/constants/constants.go

```diff
@@ -302,6 +302,9 @@ const (
 	// YAMLDocumentSeparator is the separator for YAML documents
 	// TODO: Find a better place for this constant
 	YAMLDocumentSeparator = "---\n"
+
+	// KubeletHealthzPort is the default port of kubelet healthz
+	KubeletHealthzPort = 10248
```

Comment:
this is good @xlgao-zju thanks

one small nit: please move `KubeletHealthzPort` under `KubeletEnvFileVariableName` in `constants.go`, so that we keep the kubelet related constants more compact.

please change the comment to:
> // KubeletHealthzPort is the port of the kubelet healthz endpoint

(mostly copied from the kubelet reference docs)

Reply:
Done